scripts/run-analysis.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+error() {
+  echo "$@" 1>&2
+}
+
+fail() {
+  error "$@"
+  exit 1
+}
+
+script_directory="$(perl -e 'use File::Basename;
+ use Cwd "abs_path";
+ print dirname(abs_path(@ARGV[0]));' -- "$0")"
+cd "$script_directory" || exit
+
+# However in order to give Docker access to all the code we have to
+# move up a level
+cd ..
+
+env | grep "OPENPBTA_.*" > open_pbta_envs.txt
+
+./scripts/run-analysis.sh bash download-data.sh
```

Comment:
Here are the bash commands to download OpenPBTA-analysis testing data and run `interaction-plots` module:

```suggestion
OPENPBTA_URL=https://s3.amazonaws.com/d3b-openaccess-us-east-1-prd-pbta/data OPENPBTA_RELEASE=testing bash download-data.sh

OPENPBTA_ALL=0 bash analyses/interaction-plots/01-create-interaction-plots.sh
```

I have tested these commands in a local docker container.

The `immune-deconv` module is not compatible with the downloaded OpenPBTA-analysis testing data, because it was updated in <https://github.com/PediatricOpenTargets/OpenPedCan-analysis/commit/38a71a31241c35df2bb8d82017ff7d3bcd21d6f4> to be compatible with OpenPedCan-analysis.

cc @jharenza @chinwallaa

Reply:
Ahhh! Ok, interaction plots should hopefully be a good test.

Looking at https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/.circleci/config.yml - GSVA scores or telomerase activity may also work here.

Reply:
@jharenza - Thank you for the suggestions.

I added `telomerase-activity-prediction` in <https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/173#discussion_r874167928> for more testing. I have tested it in a local Docker container.

I think `interaction-plots` and `telomerase-activity-prediction` would be sufficient for the initial placeholder testing of the CI GitHub Action.

GSVA exited with the following error:

```text
# OPENPBTA_TESTING=1 OPENPBTA_BASE_SUBTYPING=1 bash "analyses/gene-set-enrichment-analysis/run-gsea.sh"
...
Error:

ERROR: Provided input file does not exist.
Execution halted
```